.github/ISSUE_TEMPLATE.md

```diff
@@ -36,3 +21,40 @@ This is a template helping you to create an issue which can be processed as quic
     ```
  or attach as .txt or .zip file
 -->
+
+##### Issue submission checklist
+
+ - [ ] I report the issue, it's not a question
+   <!--
+   OpenCV team works with answers.opencv.org, Stackoverflow and other communities
+   to disucss problems. Tickets with question without real probelem statement will
+   be closed.
+   -->
+ - [ ] I checked the problem with documentation, FAQ, open issues,
+       answers.opencv.org, Stackoverflow, etc and have not found solution
+   <!--
+   Places to check:
+   * FAQ page: https://github.com/opencv/opencv/wiki/FAQ,
+   * OpenCV forum: https://answers.opencv.org
+   * Stackoverflow branch: https://stackoverflow.com/questions/tagged/opencv
+   -->
```

Comment:
Add this link:
`https://github.com/opencv/opencv/issues?q=is%3Aissue`

(parameter is required to show closed issues too)